### ISSUE TITLE
Questions about the code

### ISSUE THREAD

**Issue by LinjieFu-U** [ukaukaaaa/BeamDosePrediction#2]
Hello, I'm glad that you can share your work, may I ask how I should run this work, can you provide the complete code.

**Comment by ukaukaaaa** [comment]
> Hello, I'm glad that you can share your work, may I ask how I should run this work, can you provide the complete code.

Sorry for the late reply. Here is the previous reference. You can use the training code under that repo and replace the network and loss with our network and loss functions.
#1
